static/src/stylesheets/module/content/_article-immersive.scss

```diff
@@ -609,4 +620,8 @@
             margin-right: -(gs-span(5) + $gs-gutter);
         }
     }
+
+    .ad-slot--right {
+        position: static !important; // To override sticky ad
```

Comment:
I wonder whether it would make more sense to add a specific class to page types that would require this override (and have that managed in one place) rather than starting to add !important's inside specific page type CSS files?